external/js/cothority/README.md

```diff
@@ -4,20 +4,36 @@ This library offers methods to talk to a cothority node. At this point, it
 offers a socket interface that marshals and unmarshals automatically protobuf
 messages.
 
-# Usage
+## Usage
 
 Import the library using
 ```js
 import Cothority from "@dedis/cothority"
 ```
 
-Check out the example in index.html for a browser-based usage
+Check out the example in index.html for a browser-based usage.
+For typescript projects, it is easier to directly import from the subfolders:
 
-# Documentation
+```typescript
+import {DarcInstance} from "@dedis/cothority/byzcoin/contracts"
+```
+
+Avoid importing from the files in the subfolders, as these are not guaranteed
```

Comment:
Suggestion:

Do not import directly from _files_ within the subfolders, as they can be moved or renamed at any time, while the index files will always be correct.